yamllint/config.py

```diff
@@ -153,8 +153,29 @@ def validate_rule_conf(rule, conf):
         return False
 
     if isinstance(conf, dict):
-        if ('ignore' in conf and
-                not isinstance(conf['ignore'], pathspec.pathspec.PathSpec)):
+        if 'ignore-from-file' in conf and isinstance(
+            conf['ignore-from-file'], (str, list)
+        ):
+            if isinstance(conf['ignore-from-file'], str):
+                conf['ignore-from-file'] = [conf['ignore-from-file']]
+            if not (
+                isinstance(conf['ignore-from-file'], list)
+                and all(
+                    isinstance(line, str) and os.path.isfile(line)
```

Comment:
Can you remove `and os.path.isfile(line)`? Because:
\- It's not present for the sibling not-inside-a-rule error.
\- If a file doesn't exists, it will caught just a few lines later with a `FileNotFoundError`, which is better suited.

Reply:
> Can you remove and os.path.isfile(line)? 

Sure.